_posts/2017-11-20-convocatoria-editor.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Convocatoria para editor de contenidos en español de The Programming Historian 
+authors:
+- Maria Jose Afanador-Llach
+layout: post
+categories: posts
+---
+
+Desde su conformación el (equipo editorial de contenidos en español)[https://programminghistorian.org/posts/announcing-new-team-spanish-language-editors], ha traducido (28 tutoriales)[https://programminghistorian.org/es/lecciones/] del inglés al español con el apoyo de una red de colaboradores de países tales como Argentina, España, Colombia y México. Continuando con nuestro compromiso con la diversidad lingüística, The Programming Historian está buscando un nuevo miembro para contribuir al equipo de editores de contenidos en español. El editor o editora trabajará para hacer accesibles los tutoriales de PH al público hispanoparlante, facilitará los procesos de traducción de tutoriales y actuará como editor de los mismos. De igual forma apoyará las estrategias para construir una comunidad ayude a aumentar el acceso y la colaboración de PH en el mundo hispanoparlante. 
```

Comment:
Fix all links so they follow the pattern [link text](link url)

Reply:
Thanks, done!